### PR TITLE
Remove organizations

### PR DESCRIPTION
**Pull request by satococoa**
## Summary
In our new team structure, Organizations is no longer needed.

So I've removed `organizations` table and related codes.

This Pull request is based on https://github.com/giselles-ai/giselle/pull/119 branch.
We must release https://github.com/giselles-ai/giselle/pull/119 first.

## Related Issue
- https://github.com/giselles-ai/giselle/pull/119

## TODO
- [ ] migrate preview
- [ ] migrate production

### CODE REVIEW

**Comment by satococoa** [comment]
I haven't run bunx drizzle-kit migrate even on preview.
Because removing tables/columns will affect other developers working in their local environments.
Once the code review is completed, I'll run the migration.

**Review by shige — APPROVED** [review]
LGTM 🚿 

Please let me know after migrating to the preview environment, as I would like to do my operational acceptance testing as well!

**Review by toyamarinyon — APPROVED** [review]

File: migrations/0010_classy_jackpot.sql
Comment (satococoa) [inline migrations/0010_classy_jackpot.sql:1]: 🤔 
This code is generated by drizzle-kit generate.
I think dropping constraints should be done first...

I'll fix this before run migration on Preview (and development)

**Comment by satococoa** [comment]
@shige @toyamarinyon 

I have an idea about executing the migration attached to this pull request.

**Problem**
- Since we share the database with the Preview environment, dropping columns/tables will affect developers' local environments.
  - Until merge this PR, development on local environment can't continue.

**Idea**
- Merge this code first without running the migration.
- Notify developers to pull the latest main branch.
- After notifying, run the migration on preview.
- Finally, run the migration on Production.

I would appreciate your thoughts on this approach. What do you think?

**Comment by shige** [comment]
I have no problem with your plan!

**Comment by satococoa** [comment]
@shige cc/ @toyamarinyon 
Thank you!
I'll merge and deploy this now.

I'll execute migration on Preview today at 14:00 (JST).
https://github.com/giselles-ai/giselle/issues/131

**Pull request merged**